src/api/db/build.ts

```diff
@@ -1,17 +1,23 @@
 import { Build, BuildRun, Job } from './model';
 import { getLastRun } from './job';
 
-export function getBuilds(limit: number, offset: number): Promise<any> {
+export function getBuilds(userId: number, limit: number, offset: number): Promise<any> {
```

Comment:
can we make `userId?: number` optional? so admin users get all builds any way without checking the permissions later.